migration/migrator/migrations/course/20220609173520_fix_late_day_bug.py

```diff
@@ -0,0 +1,89 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
```

Comment:
I believe that there should still be a `down()` function in all migrations, regardless of whether it is a no-op -- someone feel free to correct me if this is incorrect.  Regardless, unless there's a good reason to not include one, there should be a down migration which restores the database to the previous state, just in case we ever need to revert this change at some point.

Reply:
At the meeting with @bmcutler we discussed this and I believe we concluded a down migration was not necessary. In this case, it's a function change so any code that depends on this function will use one that was updated to work correctly. I believe this is why it's not necessary to have a down migration, but we can discuss further if this is a mistake.